### PR TITLE
Avoid increasing extra vpgrs for sbcc 64 inverse

### PR DESCRIPTION
**Pull request by solaslin**
resolves #SWDEV-287877

Fix: the 2 extra vgprs reduced the occupancy of sbcc 64 inverse from 2 to 1. Change data type from size_t back to int for sbcc.
Cherry-picked from develop branch

### CODE REVIEW

**Comment by solaslin** [comment]
I updated the ticket and we'll need to wait for PMO approval before we merge this.

**Review by memmett — APPROVED** [review]

**Review by evetsso — APPROVED** [review]

**Review by feizheng10 — APPROVED** [review]

**Pull request merged**
